### PR TITLE
fix: update ui for comment input

### LINKED ISSUE #3494: [ui] : revamp comment input
**Describe the bug**
- Comment input box is small to type more than one line comment.
- We need to make the input standardise with circuitverse inputs, refer `search` input.

Screenshot from 2023-01-18 20-04-47

**Expected behavior**
Better experience to user while writing comment!

**Device Information [optional]:**
 - Browser [chrome, safari, mozilla]

**Additional context**
- If you are tweaking the UI, share the UI in the `UX` channel of our slack before raising a PR.

**Are you working on this issue? (Yes/No)**
No

### PR DESCRIPTION
**Pull request by Sauvic016**
Fixes #3494 

#### Describe the changes you have made in this PR -
- Comment input box size is changed to enhance user experience for multi-line input.

### Screenshots of the changes (If any) -
image

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

File: app/views/commontator/comments/_form.html.erb [hunk 1]
```diff
@@ -35,7 +35,7 @@
 
   <div class="field comments-field">
     <%=
-      form.text_area :body, rows: '1', id: new_record ?
+      form.text_area :body, rows: '3', id: new_record ?
```
Comment (vedant-jain03) [inline app/views/commontator/comments/_form.html.erb:38]:
```suggestion
      form.text_area :body, rows: '4', id: new_record ?
```

**Review by vedant-jain03 — CHANGES_REQUESTED** [review]
@Sauvic016 Thanks for the contribution.

If you in the `input` (refer search input), when we focus on it, we see a animations in border. Could you add similar to `textarea` as well.

File: app/views/commontator/comments/_form.html.erb (hunk 1 above)
Comment (Sauvic016) [reply app/views/commontator/comments/_form.html.erb:38]: Sure, I will make these changes asap.

**Comment by Sauvic016** [comment]
@vedant-jain03 could you review the changes once.

**Comment by Sauvic016** [comment]
@tachyons could you review the code.

**Review by vedant-jain03 — APPROVED** [review]

**Pull request merged**